src/commands/pack/win.ts

```diff
@@ -220,6 +227,9 @@ export default class PackWin extends Command {
 the CLI should already exist in a directory named after the CLI that is the root of the tarball produced by "oclif pack:tarballs"`,
       hidden: true,
     }),
+    'hide-defender-option': Flags.boolean({
```

Comment:
there's an `env` flag option that will default to what's in the environment if a value is provided. I think that would allow you to skip the env var check on line 89

Reply:
oh, it can used the scopedEnvVar thing?

Reply:
hm, yeah good point - it's not a scoped env var